certbot/cli.py

```diff
@@ -500,16 +499,23 @@ def prescan_for_flag(self, flag, possible_arguments):
             pass
         return True
 
-    def add(self, topic, *args, **kwargs):
+    def add(self, topics, *args, **kwargs):
```

Comment:
I think my prior idea was that the topics are really deep dives if you want to know about stuff that's specific to that invocation of the client, rather than (say) exhaustively listing things that are relevant to each subcommand.  Though perhaps there could also be a case for making the topic help into something more like the git manpages for subcommands, which list every possible flag for each of them?